epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.java

```diff
@@ -623,6 +656,41 @@ private void addEqualsLineForType(Builder builder, boolean useObjectHashCode, Ty
     }
   }
 
+  private void addEqualsLineForTypeUsingGetter(Builder builder, boolean useObjectHashCode,
```

Comment:
I've made it in a different way. `addEqualsLineForTypeUsingGetter` is not needed anymore.